src/utils/relayer.ts

```diff
@@ -0,0 +1,110 @@
+import { Redis } from 'ioredis'
+import { HttpErrors } from '@loopback/rest'
+import { Node } from '@pokt-network/pocket-js'
+
+import { BlockchainsRepository } from '../repositories'
+import { SyncCheckOptions } from '../services/sync-checker'
+import { DEFAULT_BLOCK_LOGS_LIMIT } from '../utils/constants'
+import { BlockchainDetails } from './types'
+
+// Fetch node client type if Ethereum based
+export async function fetchClientTypeLog(
+  redis: Redis,
+  blockchainID: string,
+  id: string | undefined
+): Promise<string | null> {
+  const clientTypeLog = await redis.get(`${blockchainID}-${id}-clientType`)
+
+  return clientTypeLog
+}
+
+export function isCheckPromiseResolved(promise: PromiseSettledResult<Node[]>): boolean {
```

Comment:
this doesn't look like is part of the relayer, it can be moved to a miscelaneus utils file